web/client/components/geostory/layouts/sections/Immersive.jsx

```diff
@@ -101,7 +101,7 @@ const Immersive = ({
                 glyph: 'book',
                 tooltipId: 'geostory.addImmersiveContent',
                 onClick: () => {
-                    add(`sections[{"id": "${id}"}].contents`, undefined, ContentTypes.COLUMN); // position undefined means append
+                    add(`sections`, 0, SectionTypes.IMMERSIVE);
```

Comment:
This creates a new immersive section below this one. Instead, as for the mock-up, we want to allow to add only contents to immersive section